src/cdk-experimental/popover-edit/edit-events.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Subject, timer} from 'rxjs';
+import {audit, distinctUntilChanged, filter, map, share} from 'rxjs/operators';
+import {Injectable} from '@angular/core';
+import {closest, CELL_SELECTOR, DEFAULT_HOVER_DELAY_MS, ROW_SELECTOR} from './common';
+
+/**
+ * Service for sharing delegated events and state for triggering edits.
+ */
+@Injectable()
+export class EditEvents {
+  readonly editing = new Subject<Element|null>();
+  readonly hovering = new Subject<Element|null>();
+  readonly mouseMove = new Subject<Element|null>();
+
+  protected currentlyEditing: Element|null = null;
```

Comment:
Do you anticipate this service being extended, if so, do you have a use case we can track to make sure we're correctly enabling that case?

Reply:
I don't have anything specific in mind. I could make them private if that's preferred.

I'll be extending a bunch of the directives for the material versions but probably not the services.